aiidalab_widgets_base/structures.py

```diff
@@ -702,33 +712,63 @@ def _rdkit_opt(self, smiles, steps):
 
         smiles = smiles.replace("[", "").replace("]", "")
```

Comment:
this was introduced to avoid problems we had with some smiles from experimentalists. Not sure what the real problem was @pgasparo should comment on this

Reply:
@cpignedoli, was it somehow related to the use of SMILES format for the GNR?